### ISSUE TITLE
Homebrew install fails on fresh OS X El Capitan (10.11)

### ISSUE THREAD

**Issue by chuckhoupt** [AppliedLogicSystems/ALSProlog#19]
Tom reported the make fails with missing X11 header for Tcl/Tk:

```
clang -m32 -std=gnu99 -O0 -MMD -Wall -Wmissing-prototypes -Wwrite-strings -Wnested-externs -Wpointer-arith -isystem /usr/X11/include -I /System/Library/Frameworks/Tk.framework/Versions/Current/Headers/ -I ../../alsp_src/port -I ../../alsp_src/unix -I ../../alsp_src/generic -I ../../alsp_src/smath  -DUNIX -DUNIX_DARWIN -DPORT  -c -o tcl_interface.o ../../alsp_src/../tcltk_interface/common/tcl_interface.c
In file included from ../../alsp_src/../tcltk_interface/common/tcl_interface.c:17:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/tk.h:78:11: fatal error: 'X11/Xlib.h' file not found
#       include <X11/Xlib.h>
                ^
1 error generated.
```

**Comment by chuckhoupt** [comment]
Couldn't reproduce in (nearly)-clean system. Check that OS has Tk Framework:

/System/Library/Frameworks/Tk.framework/Versions/Current/Headers/

**Comment by kenbowen** [comment]
IIRC, when we started all this, there was some problem one or both of us had re: presence of TclTk.

**Comment by chuckhoupt** [comment]
Fixed by fa27067a0d0debc93c8eaf055c95b886e812fc5f
